src/banim-battleparse.c

```diff
@@ -522,7 +502,7 @@ void ParseBattleHitToBanimCmd(void)
 
                     r9 = r9 + 1;
                     gEfxHpLut[r9 * 2] = new_hp;
-                    *anim_round_r5 = (u32)*anim_round_r5 | ({0xFFFF8000 + 0;});
+                    gAnimRoundData[round_sp0C * 2] |= 0x8000;
```

Comment:
Should this be [`ANIM_ROUND_DEVIL`](https://github.com/FireEmblemUniverse/fireemblem8u/blob/eda9ed07af7e52b77c68a194d3a0082f39a69b5a/include/ekrbattle.h#L287)?

Reply:
Fixed.